spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -756,6 +757,19 @@ public void open() {
       }
     }
 
+    String mvnRepo = getProperty("zeppelin.interpreter.dep.mvnRepo");
```

Comment:
I think changes in `SparkInterpreter.java` is unnecessary.
value of mvnRepo will be something like `http://repo1.maven.org/maven2/`, which doesn't need to be in the classpath.